app/helpers/form_helper.rb

```diff
@@ -60,4 +60,12 @@ def scoped_parent_activities(activity:, user:)
         .call(eager_load_parent: false)
     end
   end
+
+  def create_activity_level_options(user:)
+    authorised_levels = Activity.levels.select { |level|
+      policy = Pundit.policy(user, level.to_sym)
+      policy.create? || policy.update?
+    }
+    authorised_levels.keys.map { |level| OpenStruct.new(level: level, description: level.titleize) }
```

Comment:
I am not sure how much we should worry about 'Third-party projects' being shown but `titeize` won't do it, we use the locale elsewhere, something like:

`authorised_levels.keys.map { |level| OpenStruct.new(level: level, description: I18.t("page_content.activity.level.#{level}").capitalise) }`